### PR TITLE
media-video/obs-studio: add version 30.0.0

### PR DESCRIPTION
**Pull request by Chiitoo**

### CODE REVIEW

File: dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+inherit cmake
+
+DESCRIPTION="QR Code Generator Library in Multiple Languages"
+
+HOMEPAGE="
+	https://github.com/nayuki/QR-Code-generator
+	https://github.com/EasyCoding/qrcodegen-cmake
+"
+
+SRC_URI="
```
Comment (ionenwks) [inline dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild:15]: Typical layout is to have two blocks with only 1 blank line between them, aka:
```
DESCRIPTION=
HOMEPAGE=
SRC_URI=
S=

LICENSE=
SLOT=
KEYWORDS=
(USE=)
```
skel.ebuild does not really relay this well due to the extra splitting with comments, albeit opening an empty "file.ebuild" with e.g. vim would auto-generate a skeleton that looks like that

File: dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+LICENSE="MIT"
+
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+src_prepare() {
+	# Move the CMake files into the project root.
+	mv "${S}"/../qrcodegen-cmake-"${PV}"-cmake2/* "${S}"
```
Comment (ionenwks) [inline dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild:30]: ${S} is the current directory in src_*, and PV does not need quoting, untested but could simplify as:
```suggestion
	mv ../qrcodegen-cmake-${PV}-cmake2/* . || die
```
(was also missing || die)

File: dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild (hunk 1 above)
Comment (Chiitoo) [reply dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild:15]: Yeah, I was thinking of doing it more like that, while pondering about the skel being what it is...

File: dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild (hunk 2 above)
Comment (Chiitoo) [reply dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild:30]: Yeah, I'll test that.

Had 'pkgcheck' complaining about at least one of them unquoted being, but didn't try which one...

**Comment by ionenwks** [comment]
> I changed the masking a bit, nothing weird there?

Looks fine to me.

> Need to consider the qt 6 keyword situation still though.

Ah, that's why you were mentioning qtsvg back on IRC. Well, arm64 + x86 would be pretty trivial to do given qtbase is already done (can do a separate keywordreq), but no idea what's happening with ppc64.

**Comment by Chiitoo** [comment]
Yep, noticed qtbase is in a good shape!

Thanks again.

**Pull request closed without merging**